docs/use-github-in-bcgov/github-transition-guide.md

```diff
@@ -20,13 +20,13 @@ Our organization has many different types of IDIRs, such as:
 As we upgrade GitHub. we focus on 2 types of IDIRs:
 
 - Primary IDIRs for employees
-- Contractor IDIRs
+- Contractor IDIR
```

Comment:
I think this one works as IDIRs still? rationale: there are still different types of idiots that a contractor might have? (but maybe you're seeing something I'm not seeing)

Reply:
@jenreiher-bcgov These are type lists, so IDIR would be singular in all cases, right?  Also, LOL at that autocorrect!  :rofl:

Reply:
omg that autocorrect is making my heart SO unhappy :( :( :( sorry for not catching it!


I don't know the grammar of all of this is breaking my brain a lot. Maybe it's because IDIR is such a made up thing ahahah.